main.go

```diff
@@ -19,16 +19,31 @@ var parseCmd = &cobra.Command{
 var genCmd = &cobra.Command{
 	Use:   "generate",
 	Short: "generate golang file and print it to the output",
+}
+
+var clientCmd = &cobra.Command{
+	Use:   "client",
+	Short: "generate client golang file and print it to the output",
+	Run: func(cmd *cobra.Command, args []string) {
+		s := parse(spec)
+		render(s, "client.tmpl")
+	},
+}
+
+var serverCmd = &cobra.Command{
+	Use:   "server",
```

Comment:
i think this is not yet server, but handlers. maybe we should rename comand (and all relates stuff) to "handler"?